src/hexkit/providers/akafka/config.py

```diff
@@ -117,3 +119,15 @@ class KafkaConfig(BaseSettings):
         title="Kafka Retry Backoff",
         examples=[0, 1, 2, 3, 5],
     )
+    kafka_compression_type: Optional[KafkaCompressionType] = Field(
```

Comment:
Can you move this setting next to `kafka_max_message_size`, since they are a bit connected?

And in the `kafka_max_message_size` description, mention that this refers to the uncompressed size
and enlarging this can be combined with adding compression.